src/main/java/gov/nasa/jpl/mbee/mdk/options/MDKOptionsGroup.java

```diff
@@ -138,6 +141,21 @@ public void setUserScriptDirectory(String path) {
         property.setGroup(GROUP);
         addProperty(property, true);
     }
+    
+    
+    public String getDefaultDocBookToPDFStyleSheet() {
+    	Property p = getProperty(DEFAULT_DOCBOOK_TO_PDF_STYLESHEET);
+		return (String) p.getValue();
+    }
+    
+    public void setDefaultDocBookToPDFStyleSheet(String value) {
```

Comment:
See proposed rename above.

Reply:
Remove the Default and change to setDocBookToPDFStyleSheet...